noxfile.py

```diff
@@ -164,10 +234,10 @@ def lint_setup_py(session):
     session.run("python", "setup.py", "check", "--restructuredtext", "--strict")
 
 
-@nox.session(python="3.8")
+@nox.session(python=DEFAULT_PYTHON_VERSION)
```

Comment:
Just to clarify, do we want to run this against all versions?

Reply:
We normally only run it on 1 version but we can expand it if there is value